policy-test/tests/inbound_api.rs

```diff
@@ -34,7 +34,10 @@ async fn server_with_server_authorization() {
         // that the update now uses this server, which has no authorizations
         let server = create(&client, mk_admin_server(&ns, "linkerd-admin", None)).await;
         let config = next_config(&mut rx).await;
-        assert_eq!(config.protocol, Some(grpc::defaults::proxy_protocol()));
+        assert_eq!(
+            config.protocol,
+            Some(grpc::defaults::proxy_protocol_no_ratelimit())
+        );
```

Comment:
Preferable to not modify formatting of unrelated code -- can cause backport pain.

Reply:
It's not just formatting; `proxy_protocol_no_ratelimit` is a replacement for the `proxy_protocol` function that doesn't include rate-limiting. To minimize the change, I could instead keep `proxy_protocol` and create a new `proxy_protocol_with_ratelimiting`.